### PR TITLE
`selector-pseudo-element-case` rule

### PR DESCRIPTION
**Pull request by evilebottnawi**

### CODE REVIEW

File: docs/user-guide/rules.md [hunk 1]
```diff
@@ -139,6 +139,7 @@ Here are all the rules within stylelint, grouped by the [*thing*](http://apps.wo
 - [`selector-no-type`](../../src/rules/selector-no-type/README.md): Disallow type selectors.
 - [`selector-no-universal`](../../src/rules/selector-no-universal/README.md): Disallow the universal selector.
 - [`selector-no-vendor-prefix`](../../src/rules/selector-no-vendor-prefix/README.md): Disallow vendor prefixes for selectors.
+- [`selector-pseudo-element-case`](../../src/rules/selector-pseudo-element-case/README.md): Specify lowercase or uppercase for pseudo-elements selectors.
```
Comment (jeddy3) [inline docs/user-guide/rules.md]: "for pseudo-element selectors"

File: src/rules/selector-pseudo-element-case/README.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
+# selector-pseudo-element-case
+
+Specify lowercase or uppercase for pseudo-elements selectors.
```
Comment (jeddy3) [inline src/rules/selector-pseudo-element-case/README.md]: "for pseudo-element selectors"

**Comment by davidtheclark** [comment]
Awesome. Just needs a rebase.

**Comment by evilebottnawi** [comment]
@davidtheclark done

**Pull request merged**